src/status_im/config.cljs

```diff
@@ -48,6 +48,7 @@
 
 (def opensea-api-key OPENSEA_API_KEY)
 (def status-proxy-enabled? true)
+(def status-proxy-stage-name (get-config :STATUS_PROXY_STAGE_NAME "test"))
```

Comment:
@cammellos, I've seen many problems in other projects with using default environments. Sometimes a default in one place is not the default in the other. A classic could be having a CI issue where the var is not injected correctly and we end up pointing to the `test` stage where we shouldn't.

We can make this value mandatory and fail earlier. I didn't put much thought into the solution, and we don't have much time, so we could improve in a future PR.

Leaving here the snippet for future reference:

```clojure
(def status-proxy-stage-name
  (when status-proxy-enabled?
    (react-native-config/get-config! :STATUS_PROXY_STAGE_NAME)))

;; in react-native.config
(defn get-config!
  [k]
  (if-let [v (get config k)]
    v
    (throw (ex-info "Key not found in config" {:key k}))))
```

Reply:
Also better option I think is to only default to "test" in a debug build. In other environments it would simply throw in the absence of a valid stage name.

Reply:
makes sense, it's a bit annoying I think operationally as I am not sure `.env` files are picked up, so devs would have to set all the variables themselves when running the app locally (or use the make file, but then if we default them in the make file, we might be at the same point), but in principles I agree, is it ok if I merge this in develop and release and we revisit later?

Reply:
It's perfectly okay 👍🏼 To me it's an old debt in how we manage our configuration. It's not just this value, the `default-network` for instance also has a similar issue. Ideally we should have a robust way to inject values and eliminate as much as possible the risk of mixing values from different environments. Just scars, I've seen this happen many times.

Reply:
oh yes, we just saw it now as a matter of fact :D